tests/functional/test_install.py

```diff
@@ -1063,16 +1146,17 @@ def test_install_package_with_prefix(script, data):
     )
 
     rel_prefix_path = script.scratch / "prefix"
-    install_path = (
-        distutils.sysconfig.get_python_lib(prefix=rel_prefix_path)
-        /
+    install_path = join(
+        distutils.sysconfig.get_python_lib(prefix=rel_prefix_path),
         # we still test for egg-info because no-binary implies setup.py install
-        f"simple-1.0-py{pyversion}.egg-info"
+        f"simple-1.0-py{pyversion}.egg-info",
     )
```

Comment:
Why's this change needed?

Reply:
Reverting this causes the mypy error:

```
tests/functional/test_install.py:1163: error: Unsupported left operand type for
/ ("str")  [operator]
            distutils.sysconfig.get_python_lib(prefix=rel_prefix_path)
            ^
Found 1 error in 1 file (checked 282 source files)
```

This previous version worked in practice because `rel_prefix_path` is a `tests.lib.path.Path` object which is both a subtype of `str` and supports the `/` operator. However, mypy understands the return type of `distutils.sysconfig.get_python_lib` to be `str` and not the same type as the first argument.

Reply:
Here is the typeshed definition of `distutils.sysconfig.get_python_lib`

https://github.com/python/typeshed/blob/9b86402c77920f369c20069a6d1481a8d467c14d/stdlib/distutils/sysconfig.pyi#L12